src/Analysis/Ast/Impl/Caching/CacheFolders.cs

```diff
@@ -103,6 +100,12 @@ internal static class CacheFolders {
                 .Replace('/', '_').Replace('+', '-');
         }
 
+        /// <summary>
+        /// Given a path to a zip file, extracts it to a cache directory and returns the file path to that directory
+        /// </summary>
+        public static string GetZipCacheRoot(IOSPlatform platform)
+            => Path.Combine(GetCacheFolder(platform), "Zips/Eggs");
```

Comment:
Can use a better name if needed

Reply:
`Path.Combine` has overload with several parameters: `Path.Combine(GetCacheFolder(platform), "Zips", "Eggs")`

Reply:
I'm doing Zips/Eggs to mean a folder that contains zips and eggs. Maybe `ZipsAndEggs` is better?